src/main/java/org/symphonyoss/symphony/messageml/elements/Select.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2016-2017 MessageML - Symphony LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.symphonyoss.symphony.messageml.elements;
+
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.SelectNode;
+
+import java.util.Collections;
+
+/**
+ * Class representing dropdown menu - Symphony Elements.
+ *
+ * @author lumoura
+ * @since 3/22/18
+ */
+public class Select extends Element {
```

Comment:
Reorganize your methods, as per Java standards, the order on a file is static above non-static methods, and public -> protected -> private in this order (of appearance) as well.